cogs/displayinfo.py

```diff
@@ -33,85 +33,130 @@ class DisplayInfo(commands.Cog):
 
     def __init__(self,bot):
         self.bot = bot
-        self.timers = {}
-        self.current_collection = ""
+        self.current_collection = db["UserTracking"] # TODO: Move this secrets file
+        self.timers = self.current_collection.find_one() # Gets document from collection.
         self.current_guild_name = ""
         self.current_date = str(date.today())
+        self.members_online = set()
+        self.timed_task = None
+    
+    '''
+    Sample dictionary layout: {'_id': ObjectId('64d6c8e896c982ece6550d23'), '326586435958341636': {'2023-08-14': { 'Activity 1': {'totalTime': [], 'startTime': []}, 'Activity 2': {'totalTime': [], 'startTime': []}}}, 
```

Comment:
Should reference what this is. 
IE: `Sample dictionary layout for self.timers: {...` 

Also, things like _id, ObjectId, and long strings of numbers are not useful. We can include that but we should say what they are (like user id integer, etc.)